javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlProperties.java

```diff
@@ -5,9 +5,11 @@
 
 @ConfigurationProperties(prefix = "javers")
 public class JaversSqlProperties extends JaversSpringProperties {
+    private static final String DEFAULT_OBJECT_ACCESS_HOOK = "org.javers.hibernate.integration.HibernateUnproxyObjectAccessHook";
 
     private boolean sqlSchemaManagementEnabled = true;
     private String sqlSchema;
+    private String objectAccessHook = DEFAULT_OBJECT_ACCESS_HOOK;
```

Comment:
`ObjectAccessHook` is a core interface, it is used also in `JaversMongoAutoConfiguration`

Reply:
so this props should be added in `JaversSpringProperties` and available for both SQL and Mongo starters